backends/nxp/run_unittests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Copyright 2025 NXP
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+set -eux
+
+SCRIPT_DIR=$(dirname $(readlink -fm $0))
+EXECUTORCH_DIR=$(dirname $(dirname $SCRIPT_DIR))
+
+cd $EXECUTORCH_DIR
+
+# '-c /dev/null' is used to ignore root level pytest.ini.
+PYTHONPATH=`cd ..; pwd` pytest -c /dev/null backends/nxp/tests/
```

Comment:
Do we have to use `pytest`? Can we start with just the unit tests?

Reply:
We use pytest quite extensively - fixtures, mocking/spying, test parametrization, test skipping etc. Tests in Robert's PR are also based on pytest so it would require significant refactoring - https://github.com/pytorch/executorch/pull/10196/files#diff-5103e16cc6c48b276ba322c235272453e0442b6d19193200cd4a0e95706a605c . So if there's chance to stick with pytest, we would prefer it.

Reply:
We are trying people to discourage using pytest because it is yet-another-thing while we already have unit-tests. Is it possible to slowly migrate away from this going forward? I.e. new tests won't use pytest features etc.?

Reply:
Yes, we can go this way (new features covered by unittest). A bit sad to miss out on pytest's powers, but I understand the desire to avoid yet another dependency.

We still have some more code to upstream (beside Robert's PR) so there will be a few more PR's with pytest tests, but we're internally switching to unittest.

Reply:
@digantdesai We have replaced pytest in out newest code, but some functionality (spying) has to be substituted by library "kgb" (https://pypi.org/project/kgb/). Is combination unittest+kgb preferred then using pytest?

Reply:
@digantdesai , based on our conversation, let me refrase the question @skywall brings up. With abandoning pytest some functionality will get lost, e.g. spying. So, whether the future test shall be limited to unittest capabilities or additional packages like the kgb is OK. 
So essentially what limitation you envision for unit test creation?